src/Converter/Parsedown.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/*
+ * Copyright (c) Arnaud Ligny <arnaud@ligny.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Cecil\Converter;
+
+use Cecil\Config;
+use Cecil\Exception\Exception;
+use Cecil\Util;
+use Intervention\Image\Exception\NotReadableException;
+use Intervention\Image\ImageManagerStatic as Image;
+use ParsedownExtra;
+
+class Parsedown extends ParsedownExtra
+{
+    const TMP_DIR = '.cecil';
+    const PATTERN = '(.*)(\?|\&)([^=]+)\=([^&]+)'; // https://regex101.com/r/EhIh5N/2
+    private $config;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(Config $config = null)
+    {
+        parent::__construct();
+        $this->config = $config;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function inlineImage($excerpt)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method inlineImage() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/Narno/Cecil/pullRequest?prid=5018193)